testbed/addons/kit/construct.ts

```diff
@@ -0,0 +1,97 @@
+            apiVersion: 'v1',
+            kind: 'Namespace',
+            metadata: {
+                name: namespace
+            }
+        })
+
+        // Controller Role
+        const sa = props.cluster.addServiceAccount('kit-controller-sa', {
+            name: "kit-controller",
+            namespace: namespace
+        })
+        sa.node.addDependency(ns)
+        sa.role.attachInlinePolicy(new iam.Policy(this, 'kit-controller-policy', {
+            statements: [
+                new iam.PolicyStatement({
+                    resources: ['*'],
+                    actions: [
+                        // Write Operations
+                        "ec2:CreateTags",
+                        "ec2:CreateLaunchTemplate",
+                        "ec2:CreateLaunchTemplateVersion",
+                        "ec2:DeleteLaunchTemplate",
+                        "ec2:RunInstances",
+                        "iam:passRole",
+                        "autoscaling:CreateOrUpdateTags",
+                        "autoscaling:CreateAutoScalingGroup",
+                        "autoscaling:DeleteAutoScalingGroup",
+                        "autoscaling:UpdateAutoScalingGroup",
+                        "autoscaling:SetDesiredCapacity",
+                        //Read Operations
+                        "ec2:DescribeInstances",
+                        "ec2:DescribeLaunchTemplates",
+                        "ec2:DescribeLaunchTemplateVersions",
+                        "ec2:DescribeSubnets",
+                        "ssm:GetParameter",
+                        "autoscaling:DescribeAutoScalingGroups"]
+                }),
+            ],
+        }))
+
+        // Node Role
+        const nodeRole = new iam.Role(this, 'kit-node-role', {
+            assumedBy: new iam.ServicePrincipal('ec2.amazonaws.com'),
+            managedPolicies: [
+                iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonEKSWorkerNodePolicy'),
+                iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonEC2ContainerRegistryReadOnly'),
+                iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonEKS_CNI_Policy'),
+                iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonSSMManagedInstanceCore')
+            ]
+        })
+
+        props.cluster.awsAuth.addRoleMapping(nodeRole, {
+            username: 'system:node:{{EC2PrivateDNSName}}',
+            groups: ['system:bootstrappers', 'system:nodes']
+        })
+
+        new iam.CfnInstanceProfile(this, 'kit-instance-profile', {
+            roles: [nodeRole.roleName],
+            instanceProfileName: `KitNodeInstanceProfile-${props.cluster.clusterName}`
+        })
+
+        // Install kit
+        const chart = props.cluster.addHelmChart('kit', {
+            chart: 'kit',
+            release: 'kit',
+            version: 'v0.0.1',
+            repository: 'https://github.com/awslabs/kubernetes-iteration-toolkit/tree/main/operator/charts/kit-operator',
```

Comment:
Maybe it's ok for now, I haven't tested this repository path. 
Once we cut a release for kit-operator, we can reference the repository as `https://awslabs.github.io/kubernetes-iteration-toolkit/`